### PR TITLE
V0.14.0 audit patch

### PR DESCRIPTION
**Pull request by granular-ryanbonham**
###### Description

Fill in your description here.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/conf/logs/logs.conf [hunk 1]
```diff
@@ -5,4 +5,5 @@
 </source>
 @include logs.source.containers.conf
 @include logs.source.systemd.conf
-@include logs.source.generic.conf
\ No newline at end of file
+@include logs.source.generic.conf
+@include logs.source.kube-audit.conf
```
Comment (vsinghal13) [inline deploy/helm/sumologic/conf/logs/logs.conf]: The order should be reverse for these two. The generic filter should be at the end .

File: deploy/helm/sumologic/conf/logs/logs.source.kube-audit.conf [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<match kube-audit.**>
+    @type relabel
+    @label @KUBEAUDIT
+</match>
+<label @KUBEAUDIT>
+    <filter kube-audit.**>
```
Comment (vsinghal13) [inline deploy/helm/sumologic/conf/logs/logs.source.kube-audit.conf]: Indentation looks wrong, please correct.

File: deploy/helm/sumologic/values.yaml [hunk 3]
```diff
@@ -311,7 +311,19 @@ fluent-bit:
...
+    [INPUT]
+      Name              tail
+      Path              /var/log/kube-apiserver-audit.log
+      Parser            docker
+      Tag               audit.*
+      Refresh_Interval  5
+      Rotate_Wait       60
+      Mem_Buf_Limit     35MB
+      Buffer_Chunk_Size 2MB
+      Buffer_Max_Size   10MB
```
Comment (vsinghal13) [inline deploy/helm/sumologic/values.yaml]: @rvmiller89 @frankreno Do we want to allow this buffer limit? I would recommend keeping the same as for the container logs input section.

Comment (rvmiller89) [reply deploy/helm/sumologic/values.yaml]: +1 to keeping the same for consistency

File: deploy/helm/sumologic/conf/logs/logs.conf (hunk 1 above)
Comment (granular-ryanbonham) [reply deploy/helm/sumologic/conf/logs/logs.conf]: Fixed the ordering

File: deploy/helm/sumologic/conf/logs/logs.source.kube-audit.conf (hunk 2 above)
Comment (granular-ryanbonham) [reply deploy/helm/sumologic/conf/logs/logs.source.kube-audit.conf]: Yep tabs were set to 4 spaces, should be 2 now..

File: deploy/helm/sumologic/values.yaml (hunk 3 above)
Comment (granular-ryanbonham) [reply deploy/helm/sumologic/values.yaml]: So are you suggesting that we set the " Mem_Buf_Limit    5MB" and then remove the "Buffer_Chunk_Size" and "Buffer_Max_Size" or where do you want them set to?

Comment (granular-ryanbonham) [reply deploy/helm/sumologic/values.yaml]: Note that i got these setting from the upstream fluent bit helm chart, for what they are setting when tailing the kube-audit logs. See https://github.com/helm/charts/tree/master/stable/fluent-bit#configuration

Comment (frankreno) [reply deploy/helm/sumologic/values.yaml]: We should probably be consistent with upstream...

**Comment by samjsong** [comment]
Thanks for contributing! LGTM, we'll perform some testing internally before merging.

**Comment by frankreno** [comment]
the new fluent-bit chart changes how this works entirely.  so lets move to 2.1 and handle once we migrate to the new fluent-bit chart.

**Comment by frankreno** [comment]
Out helm chart has changed quite a lot since this PR was originally created.  There are also changes to the fluent-bit helm chart depdency.  I am closing this PR.  BUt will create a new issue to track making it easier to collect the K8s audit logs.

**Pull request closed without merging**